src/Verilator.cpp

```diff
@@ -526,6 +526,54 @@ static void process() {
     // Note early return above when opt.cdc()
 }
 
+static void execBuildJob() {
+    if (!v3Global.opt.build()) return;
+
+    std::stringstream cmd;
+    const V3StringList& makeFlags = v3Global.opt.makeFlags();
+    const int jobs = v3Global.opt.buildJobs();
+    if (v3Global.opt.gmake()) {  // If both gmake and cmake are chosen, use gmake to build.
+        cmd <<  v3Global.opt.makeCmd();
+        cmd << " -C " << v3Global.opt.makeDir();
+        cmd << " -f " << v3Global.opt.prefix() << ".mk";
+        if (jobs == 0) {
```

Comment:
Make sure jobs inits to -1.

Reply:
Check is added.
https://github.com/verilator/verilator/blob/38af6bbae435060505b40f6cc94320698c14809d/src/Verilator.cpp#L589

Reply:
Sorry, wasn't clear. What I mean is you need a way to not pass -j.  As written if buildJobs == -1 then no "-j" will get emitted.  I think opt.buildJobs should thus when constructed init to -1 so the default when no -j is given to verilator will be to not pass a -j to make.

BTW this suggests we are missing a test that uses --build without any -j.